docker-compose.yml

```diff
@@ -1,55 +1,34 @@
-#name: cwms
 volumes:
   oracle_data_1:
   auth_data:
 services:
   db:
-    # if using gvenzl/oracle-xe you'll need to adjust the environment below
-    image: ${ORACLE_DOCKER_IMAGE}
-    volumes:
-      - oracle_data_1:/opt/oracle/oradata
+    image: registry-public.hecdev.net/cwms/database-ready-ora-23.5:latest-dev
```

Comment:
You are slipping in the ready database change too?  I want that change but it seems like it should have been its own pr.  oh well.  move fast and break things I guess.

Reply:
I thought about that, but it was required to make developing this fast (having killed by existing database) so I felt it made sense to just do.